docs/classad-attributes/job-classad-attributes.rst

```diff
@@ -1455,84 +1085,52 @@ all attributes.
     be scheduled with less delay than with waiting for the
     *condor_negotiator* to match with a new machine.
 
-:index:`KillSig<single: KillSig; ClassAd job attribute>`
-:index:`job ClassAd attribute<single: job ClassAd attribute; KillSig>`
-
-``KillSig``
+:classad-attribute:`KillSig`
     The Unix signal number that the job wishes to be sent before being
     forcibly killed. It is relevant only for jobs running on Unix
-    machines. :index:`KillSigTimeout<single: KillSigTimeout; ClassAd job attribute>`
-
-:index:`job ClassAd attribute<single: job ClassAd attribute; KillSigTimeout>`
-
-``KillSigTimeout``
-    This attribute is replaced by the functionality in
-    ``JobMaxVacateTime`` as of HTCondor version 7.7.3. The number of
+    machines. :classad-attribute:`KillSigTimeout` as of HTCondor version 7.7.3. The number of
```

Comment:
The bottom line here needs a newling after "machines", and then the first two lines of `KillSigTimeout`'s definition.

Reply:
Also, if we're keeping :index: entries, this one needs one for `KILLING_TIMEOUT`.